lesson01/assignment/scripts.js

```diff
@@ -30,4 +30,89 @@ document. addEventListener("DOMContentLoaded", function(){
     inputField.value = ''
   })
 
-})
\ No newline at end of file
+  // Make the second form do a filter() operation
+  const exampleForm2 = document.querySelector('#example-form2');
+  exampleForm2.addEventListener('submit', function(event) {
+
+    event.preventDefault();
+
+    const data = [1, 3, 5, 4, 8, 2, 56, 2, -3];
+
+    const inputField = document.querySelector('#example-form2 .input');
+    const userInput = inputField.value;
+    const userNumber = parseInt(userInput);
+
+    // If the user's input was a number, then do the operation
+    if (!isNaN(userNumber)) {
+      const filteredData = data.filter(x => x >= userNumber);
+      document.querySelector('#example-form2 .output').innerHTML = filteredData;
+    }
+
+    // Clear the form field so the user can try again
+    inputField.value = '';
+  });
+
+  // Make the third form do a map() operation
+  const exampleForm3 = document.querySelector('#example-form3');
+  exampleForm3.addEventListener('submit', function(event) {
+
+    event.preventDefault();
+
+    const data = [45, 21, 99, 20, 67, 62, 75];
+
+    const inputField = document.querySelector('#example-form3 .input');
+    const userInput = inputField.value;
+    const userNumber = parseInt(userInput);
+
+    // If the user's input was a number, then do the operation
+    if (!isNaN(userNumber)) {
+      const transformedData = data.map(x => x / userNumber);
+      document.querySelector('#example-form3 .output').innerHTML = transformedData;
+    }
+
+    // Clear the form field so the user can try again
+    inputField.value = '';
+  });
+
+  // Make the fourth form do an includes() operation
+  const exampleForm4 = document.querySelector('#example-form4');
+  exampleForm4.addEventListener('submit', function(event) {
+
+    event.preventDefault();
+
+    const data = ['Billy', 'Sam', 'Luke', 'Mark', 'Alexa', 'Emily'];
+
+    const inputField = document.querySelector('#example-form4 .input');
+    const userInput = inputField.value;
+
+    const isIncluded = data.includes(userInput);
+    document.querySelector('#example-form4 .output').innerHTML = isIncluded;
+
+    // Clear the form field so the user can try again
+    inputField.value = '';
+  });
+
+  // Make the fifth form do a find() operation
+  const exampleForm5 = document.querySelector('#example-form5');
+  exampleForm5.addEventListener('submit', function(event) {
+
+    event.preventDefault();
+
+    const data = [9, 30, 1, 44, 52, 24, 85];
+
+    const inputField = document.querySelector('#example-form5 .input');
+    const userInput = inputField.value;
+
+    if (userInput === 'odd') {
```

Comment:
One thing I'd recommend is converting userinput to lowercase by doing this: `if (userInput.toLowerCase() === 'odd') {` It's usually good practice to convert the user's input incase they include a capital letter, etc.

We of course don't require you to do this, just getting extra picky given how good your code is looking!